### PR TITLE
feat: cart UI

### PR DESCRIPTION
**Pull request by harshitadaswani**
- contains a route for the cart page
- UI is divided into 2 components
- cart functionality added

### CODE REVIEW

File: src/reducer/CartReducer.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+export const initialCartState = {
+    cartItems: 0,
+    cartPrice: 0,
+    cartArr: [],
+}
+
+export const ADD_TO_CART = "ADD_TO_CART";
+export const REMOVE_FROM_CART = "REMOVE_TO_CART";
+export const DECREMENT_ITEM = "DECREMENT_ITEM";
+
```
Comment (JantuDeb) [inline src/reducer/CartReducer.jsx:12]: This is a good approach for naming `ACTION`

**Review by JantuDeb — APPROVED** [review]
Code is structured and the naming are also good. I didn't find any issues with the code.

File: src/routes/AppRoutes.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { Home, Login, Products, Signup } from "../pages";
+import { Cart, Home, Login, Products, Signup } from "../pages";
```
Comment (sushil-mod) [inline src/routes/AppRoutes.jsx:2]: I liked how you used barrel exports here .

**Review by sushil-mod — APPROVED** [review]
Overall code is nice  .  Naming convention also good . good work

**Pull request merged**